### PR TITLE
[10.0][ADD] mis_builder_budget_operating_unit: Extend budget items w/ OU-s

### PR DESCRIPTION
**Pull request by ghost**
The purpose is to extend `mis.budget.item` model w/ `operating.unit` reference  when the latter is installed, in order to construct reports for specific OU, just like it's could be done using `mis.budget.item::company_id`

### CODE REVIEW

**Comment by fclementic2c** [comment]
:+1:

**Review by sbidoul — APPROVED** [review]
LGTM

File: mis_builder_budget_operating_unit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2018 by Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'MIS Builder Budget with Operating Unit',
+    'version': '10.0.1.0.0',
+    'category': 'Reporting',
+    'author': 'Camptocamp SA, '
+              'Odoo Community Association (OCA)',
+    'license': 'LGPL-3',
```
Comment (sbidoul) [inline mis_builder_budget_operating_unit/__manifest__.py:10]: Header says AGPL, license key says LGPL. Can you check which one is correct?

File: mis_builder_budget_operating_unit/models/mis_budget_item.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+from odoo import fields, models
+
+
+class MisBudgetItem(models.Model):
+    _inherit = 'mis.budget.item'
+
+    operating_unit_id = fields.Many2one(
+        comodel_name='operating.unit',
+        string='Operating Unit',
+        # TODO: help?
```
Comment (simahawk) [inline mis_builder_budget_operating_unit/models/mis_budget_item.py]: @fclementic2c from company setting? From user? From... pick one :wink:

File: mis_builder_budget_operating_unit/views/mis_builder_budget.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+   <record id="mis_budget_item_tree_view" model="ir.ui.view">
+    <field name="name">mis.budget.item.tree</field>
```
Comment (simahawk) [inline mis_builder_budget_operating_unit/views/mis_builder_budget.xml]: keep 2 spaces indent all around

**Review by simahawk — CHANGES_REQUESTED** [review]
no readme?

File: mis_builder_budget_operating_unit/models/mis_budget_item.py (hunk 2 above)
Comment (AaronHForgeFlow) [reply mis_builder_budget_operating_unit/models/mis_budget_item.py]: I would say to leave it without any default, mis_builder_operating_unit module does not set any default to the mis.report.instances https://github.com/OCA/operating-unit/blob/10.0/mis_builder_operating_unit/model/mis_builder.py

**Comment by AaronHForgeFlow** [comment]
Hi @arkostyuk can you rebase this PR to origin/10.0? The merge commit was accidentally deleted in the base branch. Rebasing should be enough. Sorry for this.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: It works fine on runbot

**Review by simahawk — APPROVED** [review]

**Review by tschanzt — APPROVED** [review]

File: mis_builder_budget_operating_unit/README.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (sbidoul) [inline mis_builder_budget_operating_unit/README.rst:49]: For new modules, I suggest using the new README fragments: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: mis_builder_budget_operating_unit/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2018 by Camptocamp
```
Comment (grindtildeath) [inline mis_builder_budget_operating_unit/__manifest__.py:2]: Please add `# -*- coding: utf-8 -*-`

File: mis_builder_budget_operating_unit/views/mis_builder_budget.xml [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+  <record id="mis_budget_item_tree_view" model="ir.ui.view">
+    <field name="name">mis.budget.item.tree</field>
+    <field name="model">mis.budget.item</field>
+    <field name="inherit_id" ref="mis_builder_budget.mis_budget_item_tree_view"/>
+    <field name="arch" type="xml">
+      <tree position="inside">
+        <field name="operating_unit_id"/>
```
Comment (grindtildeath) [inline mis_builder_budget_operating_unit/views/mis_builder_budget.xml]: Please add `groups="operating_unit.group_multi_operating_unit"`

File: mis_builder_budget_operating_unit/models/mis_budget_item.py [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2018 by Camptocamp
```
Comment (grindtildeath) [inline mis_builder_budget_operating_unit/models/mis_budget_item.py:2]: Please add `# -*- coding: utf-8 -*-`

**Review by grindtildeath — CHANGES_REQUESTED** [review]
Small changes

**Comment by fclementic2c** [comment]
@arkostyuk @grindtildeath Can you finalize so I can merge please ? thanks

**Comment by ghost** [comment]
@grindtildeath @fclementic2c done

**Comment by grindtildeath** [comment]
@arkostyuk Can you split the README into fragments as requested by @sbidoul ?

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:

**Review by AaronHForgeFlow — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
This PR is approved, Approval not showing on top right corner as usual for some reason.

**Pull request closed without merging**